phoenix-core/src/it/java/org/apache/phoenix/query/MetaDataCachingIT.java

```diff
@@ -51,6 +51,10 @@ public class MetaDataCachingIT extends BaseTest {
     private static final Logger LOGGER = LoggerFactory.getLogger(MetaDataCachingIT.class);
     private final Random RAND = new Random(11);
 
+    private boolean isLastDDLTimestampValidationEnabled = config.getBoolean(
```

Comment:
Instead of duplicating this line in 3 different tests, do you mind creating a util method in `ValidateLastDDLTimestampUtil` something like `getValidateLastDdlTimestampEnabled(Configuration config)` similar to `getValidateLastDdlTimestampEnabled(PhoenixConnection connection)`